.eslintrc

```diff
@@ -1,12 +1,27 @@
 ---
 parser: babel-eslint
 
+plugins:
+  - react
+
 env:
   browser: true
   node: true
 
 globals:
   __DEV__: true
+  # Jest / Jasmine
+  describe: false
```

Comment:
Ah no, I just didn't realize there was a difference. I might have known back when I first did this (I think there's a test that toggles `__DEV__`)